### PR TITLE
Console release notes 2022-09-28

### PR DESCRIPTION
**Pull request by ololobus**

### CODE REVIEW

File: content/release-notes/2022-09-28.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- Console: Added `read`, `update`, `delete` project access scopes for OAuth applications.
+- Console: Added possibility to select access scopes and projects on the OAuth consent page.
```
Comment (danieltprice) [inline content/release-notes/2022-09-28.md:8]:
```suggestion
- Console: Added the ability to select access scopes and projects on the OAuth consent page.
```

File: content/release-notes/2022-09-28.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- Console: Added `protected: boolean` flag into roles public API response.
```
Comment (danieltprice) [inline content/release-notes/2022-09-28.md:9]:
```suggestion
- Console: Added a `protected: boolean` flag to the roles public API response.
```

File: content/release-notes/2022-09-28.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- Console: Changed region and platform ids from serial numbers to strings in the public API.
+- Console: Changed operations id from serial number to UUID type in the public API.
```
Comment (danieltprice) [inline content/release-notes/2022-09-28.md:11]:
```suggestion
- Console: Changed the operations id from a serial number to a UUID type in the public API.
```

File: content/release-notes/2022-09-28.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+- Control Plane: Added internal mechanism for projects relocation between pageservers.
```
Comment (danieltprice) [inline content/release-notes/2022-09-28.md:12]:
```suggestion
- Control Plane: Added an internal mechanism for relocating projects between pageservers.
```

File: content/release-notes/2022-09-28.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+- Control Plane: Stopped sending metrics to Grafana Cloud.
+- Control Plane: Removed usage of the obsolete `zenith_ctl` binary.
+- Control Plane: Split monolithic project object into project, branches and endpoints in the internal database schema.
```
Comment (danieltprice) [inline content/release-notes/2022-09-28.md:15]:
```suggestion
- Control Plane: Split he monolithic project object into project, branches, and endpoints in the internal database schema.
```

File: content/release-notes/2022-09-28.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+### Bug fixes
+
+- UI: Fixed failure after receiving the empty response in the SQL Editor.
```
Comment (danieltprice) [inline content/release-notes/2022-09-28.md:19]:
```suggestion
- UI: Fixed a failure that occurred after receiving an empty response in the SQL Editor.
```

File: content/release-notes/2022-09-28.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+- UI: Fixed an issue when switching between schemas in Tables UI did not reset the table name.
```
Comment (danieltprice) [inline content/release-notes/2022-09-28.md:20]:
```suggestion
- UI: Fixed an issue to ensure that table names are reset when switching between schemas in the Tables UI.
```

File: content/release-notes/2022-09-28.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+- Control Plane: Fixed passwordless proxy connection to just created project.
```
Comment (danieltprice) [inline content/release-notes/2022-09-28.md:21]:
```suggestion
- Control Plane: Fixed the passwordless proxy connection for the most recently created project.
```

**Review by danieltprice — APPROVED** [review]
These are great. Just very minor edits.

**Comment by ololobus** [comment]
Committed all suggestions, so I think it's ready to go and is waiting for an approval @stepashka @danieltprice

File: content/release-notes/2022-09-28.md (hunk 8 above)
Comment (danieltprice) [reply content/release-notes/2022-09-28.md]: Ho @ololobus How about:
"Fixed an issue that occurred when creating a new project using a passwordless proxy connection.

**Review by stepashka — DISMISSED** [review]

File: content/release-notes/2022-09-28.md (hunk 8 above)
Comment (ololobus) [reply content/release-notes/2022-09-28.md]: Sounds good to me! Reworded a bit, though

**Pull request merged**